assets/scss/app.scss

```diff
@@ -1,16 +1,25 @@
 @import "~bootstrap/dist/css/bootstrap.css";
 @import "~font-awesome/scss/font-awesome";
+
+// Basic AdminLTE styles without any plugins,
+// because they conflict with select2 v4
 @import "~admin-lte/dist/css/alt/AdminLTE-without-plugins.css";
 @import "~icheck/skins/square/blue.css";
 
 // Eonasdan Bootstrap DateTimePicker in its version 3 does not
 // provide the scss or plain css, it only provides the less version
 // of its source files, that's why it is not included it via npm.
 @import "../vendor/bootstrap-datetimepicker.css";
-@import "../vendor/jquery-ui.css";
+
+// Only use sortable widget from jquery-ui library
```

Comment:
We should try to be consistent in the way we refer these packages, using the dependency name ([`jquery-ui`](https://www.npmjs.com/package/jquery-ui)) or its proper noun ([jQuery UI](https://jqueryui.com/)).

Reply:
Maybe it is better to reference using the noun for all packages, wdyt?

Reply:
Done